cg/store/models.py

```diff
@@ -992,7 +992,7 @@ class RunDevice(Base):
     )
 
     @property
-    def _samples(self) -> list[Sample]:
+    def samples(self) -> list[Sample]:
```

Comment:
I just made this "unprotected" so that pycharm won't complain. If is necessary to set it as protected, let me know

Reply:
this this is fine